Lekce_05_BR01/Program.cs

```diff
@@ -131,9 +128,15 @@ public static void Main()
         };
 
             // 7. Řešení
-            // najít milionáře a pak dát grouping podle kodu banky
 
-            List<SkupinaMilionaru> skupinyPodleBanky = null;
+            List<SkupinaMilionaru> skupinyPodleBanky = zakaznici
+                .Where(x => x.Zustatek >= 1000000)
```

Comment:
nit: Pro prehlednost se velka cisla pisou takhle :) Bude jednodussi poznat ze je to milion. Na funkcionalitu to ale nema zadny dopad.
Taky lambda by se mela jmenovat podle prvniho pismena toho objektu (v tomhle z jako Zakaznik)
```suggestion
                .Where(z => z.Zustatek >= 1_000_000)
```